contrib/sssd.spec.in

```diff
@@ -188,7 +189,8 @@ Requires: (sssd-nfs-idmap = %{version}-%{release} if libnfsidmap)
 Requires: libsss_idmap = %{version}-%{release}
 Requires: libsss_certmap = %{version}-%{release}
 %if 0%{?rhel}
-Requires(pre): shadow-utils
+BuildRequires:  systemd-rpm-macros
```

Comment:
Why is it BuildRequires (and not Requires)?

Reply:
This, and several of your other concerns, are explained in https://docs.fedoraproject.org/en-US/packaging-guidelines/UsersAndGroups/#_allocation_strategies

Reply:
Guide says "add a BuildRequires" but doesn't actually *explain*.